app/components/markdown/index.js

```diff
@@ -303,9 +308,29 @@ export default class Markdown extends PureComponent {
         );
     }
 
+    renderEditedIndicator = ({context}) => {
+        let message = '';
+        let opacity = Platform.select({ios: 0.7, android: 1});
+        if (context[0] === 'paragraph') {
+            message = ' ';
+            opacity = Platform.select({ios: 0.5, android: 0.6});
+        }
+        message += this.props.intl.formatMessage({id: 'post_message_view.edited', defaultMessage: '(edited)'});
+        return <Text style={[style.editedIndicatorText, {opacity}]}>{message}</Text>;
```

Comment:
Here you can use the `FormattedText` component that will do the localization for you, and then you can get rid of the intl prop